include/server/api/route_parameters_grammar.hpp

```diff
@@ -50,17 +57,25 @@ struct RouteParametersGrammar : public BaseParametersGrammar<Iterator, Signature
             "full", engine::api::RouteParameters::OverviewType::Full)(
             "false", engine::api::RouteParameters::OverviewType::False);
 
+        annotations_type.add("true", engine::api::RouteParameters::AnnotationsType::All)(
+            "false", engine::api::RouteParameters::AnnotationsType::None)(
+            "duration", engine::api::RouteParameters::AnnotationsType::Duration)(
+            "nodes", engine::api::RouteParameters::AnnotationsType::Nodes)(
+            "distance", engine::api::RouteParameters::AnnotationsType::Distance)(
+            "weight", engine::api::RouteParameters::AnnotationsType::Weight)(
+            "datasources", engine::api::RouteParameters::AnnotationsType::Datasources);
```

Comment:
Hm we need a mapping `AnnotationsType` -> `string` here, too. The same is needed for the response generation (as commented above). Maybe we should add this mapping already next to the `AnnotationsType` enum class definition. This would guarantee that we're in sync with name and enum item.